src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/client/WorkforceAllocationsToDeliusApiClient.kt

```diff
@@ -86,6 +87,12 @@ class WorkforceAllocationsToDeliusApiClient(private val webClient: WebClient) {
           else -> throw response.createExceptionAndAwait()
         }
       }
+
+  suspend fun personOnProbationStaffDetails(crn: String, staffCode: String): PersonOnProbationStaffDetailsResponse = webClient
+    .get()
+    .uri("/allocation-demand/impact?crn=$crn&staff=$staffCode")
```

Comment:
the endpoint is used by the impact page as well, hence the name. I thought to keep the method the name of the context hmpps-allocations is using it for